themes/website/static/js/osdetect.js

```diff
@@ -34,17 +34,29 @@ if (window.navigator.userAgent.indexOf("Mac") != -1) {
 	OSName = "Mac";
 
 	clients = [{
-		title: "qTox (Apple Sillicon)",
+		title: "qTox (macOS 12+, Apple Sillicon)",
```

Comment:
"silicon" w/ 1 L. (also in more places in the PR)

Reply:
Done.